config.json

```
@@ -611,6 +611,24 @@
                 "data_structures",
                 "algorithms"
             ]
+        },
+        {
+            "uuid": "4fc89443-1c5c-492a-850f-7990869caae6",
+            "slug": "perfect-numbers",
+            "core": true,
```

```suggestion
            "core": false,
```

I'll have to think about where this fits into the progression, let's add it as floating exercise for now.